src/components/Navigation/index.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import styled from 'styled-components';
+
+import { BadgeProps } from '../../Badge';
+import { media } from '../../BreakPoints';
+import { gray100 } from '../../Colors';
+import { IconProps } from '../../Icon';
+import { HTMLDivProps } from '../../interfaces/props';
+import NavigationSection from './NavigationSection';
+
+export interface NavigationSectionAction {
+  onClick: () => any;
+  icon: React.ReactElement<IconProps>;
+}
+
+export interface NavigationSectionSubItem {
+  url: string;
+  label: string;
+  badge?: string | React.ReactElement<BadgeProps>;
+}
+
+export interface NavigationSectionItem {
+  url?: string;
+  label: string;
+  icon: React.ReactElement<IconProps>;
+  badge?: string | React.ReactElement<BadgeProps>;
+  subItems?: NavigationSectionSubItem[];
+}
+
+interface Props extends HTMLDivProps {
+  location?: string;
```

Comment:
HTMLDivProps는 extends 하지않고 {element}Attributes로 따로 내려받게 해주세요
```suggestion
interface Props {
  location?: string;
  className?: string;
  divAttributes: HTMLDivProps;
```
history: https://github.com/pedaling/class101-ui/pull/25#discussion_r262352472

Reply:
수정하였습니다. Nav가 없어서 Div로 했슴다